homework/index.js

```diff
@@ -30,13 +30,127 @@
     return elem;
   }
 
+  function buildRepositoryInfo(repositoryObject, containerDiv) {
+    const repositoryTable = createAndAppend('table', containerDiv, {
+      id: 'repository-table',
+    });
+    const repositoryTableBody = createAndAppend('tbody', repositoryTable, {});
+    const repositoryTableRow1 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow1, { text: 'Repository:' });
+    const tableName = createAndAppend('td', repositoryTableRow1, {});
+    createAndAppend('a', tableName, {
+      text: repositoryObject.name,
+      href: repositoryObject.html_url,
+      target: '_blank',
+    });
+    const repositoryTableRow2 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow2, { text: 'Description:' });
+    createAndAppend('td', repositoryTableRow2, { text: repositoryObject.description });
+    const repositoryTableRow3 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow3, { text: 'Forks:' });
+    createAndAppend('td', repositoryTableRow3, { text: repositoryObject.forks_count });
+    const repositoryTableRow4 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow4, { text: 'Last updated:' });
+    createAndAppend('td', repositoryTableRow4, { text: repositoryObject.updated_at });
```

Comment:
1. There is some code repetition here, that could be extracted into a function, say, named `addRow`.
2. The date format presented to the user could be a bit more user friendly, by using this method on new Date(repository.updated_at): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString